test/services/notifications/setup/batch-notifications.service.test.js

```diff
@@ -367,6 +369,30 @@ describe('Notifications Setup - Batch notifications service', () => {
         }
       ])
     })
+
+    it('should update the "event.metadata.error"', async () => {
+      await BatchNotificationsService.go(testRecipients, determinedReturnsPeriod, referenceCode, journey, eventId)
+
+      const updatedResult = await EventModel.query().findById(eventId)
+
+      expect(updatedResult.metadata.error).to.equal(1)
+
+      expect(updatedResult).to.equal({
+        createdAt: updatedResult.createdAt,
```

Comment:
I hadn't spotted it on this one.

```suggestion
        createdAt: event.createdAt,
```